kubernetes/check-pods.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash -e
+
+pod_label=$1
+statefulset=$2
+
+# Statefulset starts the pods individually, but with the convention `{statefulset-name}-{0-n}`
+# The second member {0-n} corresponds to the amount of pods
+# We have to manually defined name of the pods based of amount of replicas.
+if [[ "$statefulset" == "true" ]]; then
+    pods_name=()
+    for i in `seq 0 $(($(kubectl get statefulset $pod_label -o jsonpath='{.spec.replicas}')-1))`; do
+        pods_name+=(pods/$pod_label-$i)
+    done
+    [ -z "$pods_name" ] && echo "The pod does not exist!" && exit 1
+else
+    pods_name=$(kubectl get pod -l "run=$1" -o name)
+
+    [ -z "$pods_name" ] && echo "The pod does not exist!" && exit 1
+
+    # Divide string to array by newline characters
+    readarray -t pods_name <<< "$pods_name"
+fi
+
+for i in "${pods_name[@]}"; do
+    timeout              \
+        --verbose        \
+        70               \
```

Comment:
Please make the timeout a parameter.

Reply:
Could you be more specific ? You want an input parameter that contains the time or the regular parameter.

Reply:
I want it to be a parameter of the `check-pods.sh` script so that you can do:

``` bash
./check-pods.sh rabbitmq false 120
```

or, even better (you could use something like [getopt](http://aplawrence.com/Unix/getopts.html) for parsing arguments):
``` bash
./check-pods.sh rabbitmq --stateful-set=false --timeout=120
```

Reply:
Done.